src/app/feedback/[id]/page.tsx

```diff
@@ -0,0 +1,52 @@
+"use client";
+
+import Sidebar from "@/components/sidebar/sidebar";
+import { getBookingInfo, getBoatInfo } from "@/services/notion.service";
+import { useEffect, useState } from "react";
+import router from "next/router";
+import FeedbackForm from "@/components/feedback";
+
+export default function UserFeedbackPage({ params }: { params: { id: string } }) {
+  const [boatInfo, setBoatInfo] = useState<any>({});
+  const [loading, setLoading] = useState<boolean>(true);
+  const [data, setData] = useState<any>({});
+
+  const getBoatDetails = async (data: any) => {
+    await getBoatInfo(data.Boat[0]).then((data) => {
+      setBoatInfo(data);
+    });
+  };
```

Comment:
I don't understand the logic behind these small functions that are calling to the API.

This can go inside the use effect directly 

```
const data =   await getBoatInfo(data.Boat[0]);
setBoatInfo(data);
  ````
    
    No need the the small wrapper function